packages/documentation/README.md

```diff
@@ -1,25 +1,51 @@
-# Docusaurus
-
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
-
-### Installation
+# Starlight Starter Kit: Basics
 
 ```
-$ pnpm i
+npm create astro@latest -- --template starlight
 ```
 
-### Local Development
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/withastro/starlight/tree/main/examples/basics)
+[![Open with CodeSandbox](https://assets.codesandbox.io/github/button-edit-lime.svg)](https://codesandbox.io/p/sandbox/github/withastro/starlight/tree/main/examples/basics)
 
-```
-$ pnpm start
-```
+> 🧑‍🚀 **Seasoned astronaut?** Delete this file. Have fun!
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+## 🚀 Project Structure
 
-### Build
+Inside of your Astro + Starlight project, you'll see the following folders and files:
 
 ```
-$ pnpm build
+.
+├── public/
+├── src/
+│   ├── assets/
+│   ├── content/
+│   │   ├── docs/
+│   │   └── config.ts
+│   └── env.d.ts
+├── astro.config.mjs
+├── package.json
+└── tsconfig.json
 ```
 
-This command generates static content into the `build` directory and can be served using any static contents hosting service.
+Starlight looks for `.md` or `.mdx` files in the `src/content/docs/` directory. Each file is exposed as a route based on its file name.
+
+Images can be added to `src/assets/` and embedded in Markdown with a relative link.
+
+Static assets, like favicons, can be placed in the `public/` directory.
+
+## 🧞 Commands
+
+All commands are run from the root of the project, from a terminal:
+
+| Command                   | Action                                           |
+| :------------------------ | :----------------------------------------------- |
+| `npm install`             | Installs dependencies                            |
```

Comment:
Let's have this be `pnpm`. 
Having pnpm be used instead should also end up in eliminating the `package-lock.json` file

Reply:
Sorry this file is totally not updated from default. Will make sure to use pnpm for the commands.